crawl-ref/source/dat/des/portals/dreams.des

```diff
@@ -0,0 +1,231 @@
+##############################################################################
+#
+# The realm of dreams! (amcnicky)
+#
+# A visit here is the player's reward for finding and using the dreamshard
+# necklace.
+#
+# The twofold intentions of the portal are:
+# 1. to give the player a somewhat safe on-demand (but one-time use) portal
+#    that they can step into as a very strong 2-turn escape like a ziggurine.
+#    (though it is harder to rest on the spot than in zig:1)
+# 2. to provide a good level of loot (less than a trove, more than the
```

Comment:
It's a good question - what does the loot add to the item? I think it gives the player something to 'play' in the first few dreams - i.e. what can they collect without getting netted/butchered? Thereafter, the loot arguably adds less to the experience, although I do quite like the brain worm acquiring 'knowledge' (books) from within the brain dream. Perhaps the direction is to tone down the loot a little based on play-testing (open to balance suggestions on that!), but in lieu of a reason for why loot on this item makes the item worse for crawl in general, I think the presence of loot adds rather than detracts from the design.